tests/FakeItEasy.Specs/CallMatchingSpecs.cs

```diff
@@ -732,23 +732,18 @@ public static void UnusedArgumentMatcherDescriptionNotUsed(IHaveNoGenericParamet
         public static IEnumerable<object[]> Fakes()
         {
             yield return new object[] { A.Fake<object>(), "Faked " + typeof(object) };
-            yield return new object[] { A.Fake<List<int>>(), "Faked " + typeof(List<int>) };
-            yield return new object[] { A.Fake<IList<int>>(), "Faked " + typeof(IList<int>) };
+            yield return new object[] { A.Fake<IList<int>>(), "[]" };

```

Comment:
I'm not entirely sure about rendering as `[]`. We _could_ add a formatter that detected fakes and render as `Faked Blah`. I'm not sure which is better. Honestly, I'd hope that there aren't too many Fakes being passed around, but continue to note that it's a Fake could still be helpful. Thoughts?

Reply:
> We could add a formatter that detected fakes and render as Faked Blah

Actually we can't. Formatters are selected based on their `ForType` property, and this doesn't let us detect if an object is a fake. We would need formatters to have `CanFormat` method, but that would be a breaking change.

I think there's little value in knowing that a given collection is actually a fake. But if we really want to do it, we could modify the `DefaultEnumerableValueFormatter` to check if it's a fake, and add a `Faked ` prefix to the collection description.

Reply:
> Actually we can't.